packages/video_player/example/test_driver/video_player.dart

```diff
@@ -0,0 +1,19 @@
+import 'dart:async';
```

Comment:
revert

Reply:
oops! How funny that this is included in this PR